cmd/crictl/container.go

```diff
@@ -111,6 +114,17 @@ var createPullFlags = []cli.Flag{
 		Value: "",
 		Usage: "Use `USERNAME` for accessing the registry. The password will be requested on the command line",
 	},
+	&cli.DurationFlag{
+		Name:    "cancel-timeout",
+		Aliases: []string{"T"},
+		Usage:   "Seconds to wait for a container create request to complete before cancelling the request",
```

Comment:
should it say - "disabled if 0" as well?

Reply:
or anything about the default

Reply:
The help mentions the default value:

```
   --pull-timeout value, --pt value                               Maximum time to be used for pulling the image, disabled if set to 0s (default: 0s) [$CRICTL_PULL_TIMEOUT]
```